app/components/buttons/toggle-button.jsx

```
@@ -0,0 +1,38 @@
+ */
+
+export default function ToggleIconButton(props) {
+  const [active, setActive] = useState(false)
+
+  return (
+    <div className="toggle-icon-btn" role="button" onClick={() => {
+      setActive(!active)
+      if (props.onClick) {
+        props.onClick()
+      }
+    }}>
```

Biome (1.9.4)

[error] The HTML element with the interactive role "button" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "button".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)